analyses/cellranger-analysis/submit-multiple-jobs.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+#BSUB -P run_cellranger
+#BSUB -J submitter
+#BSUB -q standard
+#BSUB -n 1
+#BSUB -R "rusage[mem=2500]"
+#BSUB -o submitter.out
+#BSUB -e submitter.err
+#BSUB -cwd "/research/dept/dnb/core_operations/Bioinformatics/achroni/GitHub/sc-rna-seq-snap/analyses/cellranger-analysis"
+
+########################################################################
+# Set up running directory
+prefix="/research/dept/dnb/core_operations/Bioinformatics/achroni/GitHub/sc-rna-seq-snap/analyses/cellranger-analysis"
+cd "$(dirname "${BASH_SOURCE[0]}")" 
+########################################################################
+# Create directories to save output files to
+mkdir -p ${prefix}/input
+mkdir -p ${prefix}/results
+mkdir -p ${prefix}/results/01_logs
+mkdir -p ${prefix}/results/02_cellranger_count
+mkdir -p ${prefix}/results/02_cellranger_count/DefaultParameters
```

Comment:
Would this need to be edited for runs that don't use the default parameters to create an output folder besides /DefaultParameters? Or will CellRanger automatically make the --output_dir from j1.sh if it doesn't exist? 

If the file path does need to exist already, maybe include a commented out command for making another results directory and include instructions to edit that in the comments in this script and in the README instructions where other script edits for runs non-default parameters are discussed

Reply:
If the user wants to run the code with other parameters and compare, they will only need to rename this folder. We will do this in the next PR after we finish reviewing the code, i.e., run the module under different parameters.